src/themes/createDarkTheme.ts

```diff
@@ -231,6 +231,27 @@ export function createDarkTheme(): Theme {
           },
         },
       },
+      CodeEditor: {
+        theme: {
```

Comment:
Can we call this property `monacoTheme` to be more specific? There are different kinds of themes at play here (monaco, shiki and mui)